tools/scripts/generate.sh

```diff
@@ -35,3 +35,4 @@ npm exec openapi-generator-cli -- generate \
     --package-name="$client_package" \
     --ignore-file-override=config/.go-ignore
 
+gofmt -s -w "$SDK_FOLDER/"*.go
```

Comment:
Ensuring that generate always have output that is compatible with gofmt and golangcilint. 

This unifies CI/CD use cases with local dev.